web/sites/default/config/field.field.taxonomy_term.council_committee.field_vice_chair.yml

```diff
@@ -0,0 +1,34 @@
+uuid: 43f84e86-a6e0-42da-8bf1-10919bb378b5
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.taxonomy_term.field_vice_chair
+    - group.type.elected_official
+    - taxonomy.vocabulary.council_committee
+  module:
+    - tmgmt_content
+third_party_settings:
+  tmgmt_content:
+    excluded: false
+id: taxonomy_term.council_committee.field_vice_chair
+field_name: field_vice_chair
+entity_type: taxonomy_term
+bundle: council_committee
+label: 'Vice Chair'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:group'
+  handler_settings:
+    target_bundles:
+      elected_official: elected_official
```

Comment:
If we can use the `Voting Electeds` entity reference view on these 3 chair/vice/members fields, we could trim down the list so auditors and old commissioners don't show up.

![image](https://github.com/user-attachments/assets/554bfe01-c000-496b-8435-c1cab9bc2577)


![image](https://github.com/user-attachments/assets/a01346ba-e268-4c6a-b286-b9110aa70760)